lib/is.js

```diff
@@ -172,17 +172,39 @@ var util = require('./util');
  * Twitch tests.
  */
 (function () {
-  var name = '[a-z0-9]\\w{3,24}';
-  def({
-      fn: 'twitch.name',
-      re: new RegExp('^' + name + '$', 'i')
-  });
-  def({
-      fn: 'twitch.url',
-      re: url('twitch\\.tv\\/' + name)
-  });
+    var name = '[a-z0-9]\\w{3,24}';
+    def({
+        fn: 'twitch.name',
+        re: new RegExp('^' + name + '$', 'i')
+    });
+    def({
+        fn: 'twitch.url',
+        re: url('twitch\\.tv\\/' + name)
+    });
 })();
 
+/**
+ * Medium tests.
+ */
+(function () {
+    var handle = '[\\w.]{1,30}';
+    def({
+        fn: 'medium.handle',
+        re: new RegExp('^@?' + handle + '$'),
+        after: function (data, options) {
+            if (options.at == null) {
+                return true;
+            }
+
+            return (options.at && data[0] === '@') || (!options.at && data[0] !== '@');
```

Comment:
You could shorten this if you want

```suggestion
            return options.at !== (data[0] === '@');
```

Reply:
Good call on shortening it up. It should be
```
return options.at === (data[0] === '@');
```